app/models/concerns/remotable.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Remotable
+  include HttpHelper
+  extend ActiveSupport::Concern
+
+  included do
+    attachment_definitions.each_key do |attachment_name|
+      attribute_name = "#{attachment_name}_remote_url".to_sym
+      method_name = "#{attribute_name}=".to_sym
+
+      define_method method_name do |url|
```

Comment:
It's a bit late, but I have just discovered that this does not overwrite the Paperclip method, so this code is never used.